### PR TITLE
Feature/arrow component

### PR DESCRIPTION
**Pull request by hansbotnen**
Arrows can be point in each cardinal direction and be directional, bidirectional or nondirectional.

### CODE REVIEW

File: src/components/Arrow/Arrow.test.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+      <Arrow
+        start={{ x: 0, y: 0 }}
+        end={{ x: 100, y: 0 }}
+        color="#000000"
+        type={ArrowType.Directional}
+      />,
+    ).container;
+
+    expect(arrow.querySelector("div")).toBeTruthy();
+  });
```
Comment (boyum) [inline src/components/Arrow/Arrow.test.tsx:17]: Would it make sense to add a few more tests to test the SVG rendering? I don't think we'd need any more than these.

```suggestion
  it("should render", () => {
    const arrow = render(
      <Arrow
        start={{ x: 0, y: 0 }}
        end={{ x: 100, y: 0 }}
        color="#000000"
        type={ArrowType.Directional}
      />,
    ).container;

    expect(arrow.querySelector("div")).toBeTruthy();
    expect(arrow.querySelectorAll("svg").length).toBe(2);
  });
  
  it("should add two arrow heads if the arrow is bidirectional", () => {
    const arrow = render(
      <Arrow
        start={{ x: 0, y: 0 }}
        end={{ x: 100, y: 0 }}
        color="#000000"
        type={ArrowType.BiDirectional}
      />,
    ).container;

    expect(arrow.querySelector("div")).toBeTruthy();
    expect(arrow.querySelectorAll("svg").length).toBe(3);
  });
  
  it("should render", () => {
    const arrow = render(
      <Arrow
        start={{ x: 0, y: 0 }}
        end={{ x: 100, y: 0 }}
        color="#000000"
        type={ArrowType.NonDirectional}
      />,
    ).container;

    expect(arrow.querySelector("div")).toBeTruthy();
    expect(arrow.querySelectorAll("svg").length).toBe(1);
  });
```

File: src/components/Arrow/Arrow.module.scss [hunk 2]
```diff
@@ -0,0 +1,32 @@
+.arrow {
+  width: 100px;
+  height: 100px;
+  box-sizing: border-box;
```
Comment (boyum) [inline src/components/Arrow/Arrow.module.scss:4]: Is explicitly setting `box-sizing` necessary here?

**Review by boyum — APPROVED** [review]
Looks good 🎉 Please look through my comments and do as you wish with them.

File: src/components/Arrow/Arrow.module.scss (hunk 2 above)
Comment (hansbotnen) [reply src/components/Arrow/Arrow.module.scss:4]: This was added earlier and turns out it was not needed. I have removed it.

**Review by boyum — APPROVED** [review]

**Pull request merged**